californium-core/pom.xml

```diff
@@ -37,6 +37,35 @@
 	<build>
 		<plugins>
 			<plugin>
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<extensions>true</extensions>
+				<configuration>
+					<instructions>
+						<Export-Package>
+							org.eclipse.californium.core,
```

Comment:
We cannot depend on artifacts that are only available from the Eclipse repo. All our dependencies must be available from Maven Central.
Regarding netty, we could replace the dependency on netty-all with dependencies for the individual artifacts.